packages/lesswrong/server/callbacks/votingCallbacks.ts

```diff
@@ -15,7 +15,7 @@ import { createAdminContext } from '../vulcan-lib/query';
 import Tags from '../../lib/collections/tags/collection';
```

Comment:
Github won't let me leave a comment on the correct line, but the `Comments` import on line 13 is now unused